src/java.base/share/classes/com/sun/crypto/provider/JceKeyStore.java

```diff
@@ -99,7 +99,7 @@ private static final class TrustedCertEntry {
      * Private keys and certificates are stored in a hashtable.
      * Hash entries are keyed by alias names.
      */
-    private Hashtable<String, Object> entries = new Hashtable<String, Object>();
+    private final Hashtable<String, Object> entries = new Hashtable<String, Object>();
```

Comment:
If the line is longer than 80 chars, the convention is to break it into 2 lines.